Add ATAC (#41)
ratatui-org/awesome-ratatui: README.md

- [TaskUI](https://github.com/thmshmm/taskui) - Simple Terminal UI for Task / taskfile.dev.
- [tenere](https://github.com/pythops/tenere) - TUI interface for LLMs written in Rust.
- [Yazi](https://github.com/sxyazi/yazi) - Blazing fast terminal file manager written in Rust, based on async I/O.

> - [ATAC](https://github.com/Julien-cpsn/ATAC) - A feature-full TUI API client made with the lovely ratatui. ATAC is free, open-source, offline and account-less.

Reviewer: Let's simplify the description a bit 🐻 

```suggestion
- [ATAC](https://github.com/Julien-cpsn/ATAC) - A feature-full TUI API client for your terminal.
```

Author: Thank you!

> - [ATAC](https://github.com/Julien-cpsn/ATAC) - A feature-full TUI API client made with the lovely ratatui. ATAC is free, open-source, offline and account-less.

Reviewer: ```suggestion
- [ATAC](https://github.com/Julien-cpsn/ATAC) - A feature-full TUI API client for your terminal.
```

Reviewer: Can you accept my suggestion? 👀
